androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -166,6 +177,15 @@ fun EnhancedVideoPlayer(
                 }
             }
         )
+
+        SeekHandler(
+            toggleIsControlsVisible = { isControlsVisible = !isControlsVisible },
```

Comment:
Some functions you are defining in the composable call and others in the `EnhancedVideoPlayer` component: `toggleIsControlsVisible` and `setIsControlsVisible` for example.

Is there a specific reason for that? I'd vouch for us to pick one of them

Reply:
I created the two functions because there are cases where I just want to toggle the variable value and there are cases where I explicitly need to set the variable value to false, I need to make a decision or pass the current value of the variable and add the logic inside the child component or create a function that already does this in the parent element.

Reply:
So it's better to pass just the `setIsControlsVisible` and create a function with the toggle logic in the `SeekHandler` component